### PR TITLE
chore(client): switch dropdown from react-select to headless UI

### PR DESCRIPTION
**Pull request by Akirtovskis**
Closes : https://github.com/ledokku/ledokku/issues/168

image

### CODE REVIEW

File: client/src/ui/components/Button.tsx
Comment (Akirtovskis) [inline client/src/ui/components/Button.tsx:21]: @pradel I added this prop because when disabled button was with `opacity-50`, it was appearing in front of dropdown menu. 

For all the other disabled cases this opacity way works fine, but for the case below dropdown it didn't not so therefore the hacky way. 

Do you see any better way this could be fixed?

File: client/src/pages/app/index.tsx
Comment (pradel) [inline client/src/pages/app/index.tsx:320]: this can be removed

**Review by pradel — APPROVED** [review]
Good to go, if you can just remove the comment before merging, otherwise feel free to merge 🚀

**Pull request merged**
